pkgs/development/tools/turso-cli/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+buildGoModule rec {
+  pname = "turso-cli";
+  version = "0.79.0";
+
+  src = fetchFromGitHub {
+    owner = "tursodatabase";
+    repo = "turso-cli";
+    rev = "v${version}";
+    hash = "sha256-5ucStAFe3lZgnGMI0fRw1E4T60+9nglNbZnzrjRmRgk=";
+  };
+
+  vendorHash = "sha256-+F9I6+f7Sm5qhBAoXCMKjV/jFY0fyVIk0NKBQNNI+qM=";
+
+  doCheck = false;
```

Comment:
Why the tests are disabled?

Reply:
`Test_setDatabasesCache` fails due to `/homeless-shelter: read-only file system error`. The test uses `configdir.MakePath` to try to make `/homeless-shelter/Library/Application` `Support/turso`. If there is a workaround for this that I am missing please let me know :)

Reply:
Ok, can you please add a comment in the pr explaining in one sentence why they are disabled?